admin/src/views/UserSearch.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <label>Usersuche</label>
+    <b-input
+      type="text"
+      v-model="criteria"
+      class="shadow p-3 mb-5 bg-white rounded"
+      placeholder="User suche"
```

Comment:
localise

Reply:
ist auf dem schirm, wird zu einem späteren zeitpunkt in einem abwasch gemacht.